app/js/HelloWorldSceneAR.js

```diff
@@ -56,6 +56,7 @@ export default class HelloWorldSceneAR extends Component {
   _onInitialized(state, reason) {
     if (state == ViroConstants.TRACKING_NORMAL) {
       this.setState({
+        // CR * what do?
```

Comment:
This tracking indicator seems useful, but what does `text: 'Color?'` end up doing??